yarn-project/end-to-end/src/e2e_cross_chain_messaging.test.ts

```diff
@@ -245,11 +246,16 @@ describe('e2e_cross_chain_messaging', () => {
       secretHashForL2MessageConsumption,
     );
 
+    // get message leaf index, needed for claiming in public
+    const maybeIndexAndPath = await aztecNode.getL1ToL2MessageMembershipWitness('latest', msgHash, 0n);
+    assert(maybeIndexAndPath !== undefined, 'Message not found in tree');
```

Comment:
Should this be an "expect" instead of an "assert"? I'm not used to seeing asserts in our tests.

Reply:
I kind of interpreted it as a precondition, but you are right might be better with expect. Changed.